### DISCUSSION TITLE
Migrated Sentry from Legacy to 6.7.0 - logger.error not pushing events to Sentry

### DISCUSSION THREAD

**Discussion by ravisaluru** [getsentry/sentry-java#2370] [Q&A]
Recently we moved away from the dropwizard-sentry (with logback) to the Sentry 6.7.0. Earlier, when we captured logs at the error level, the events were automatically sent to Senty, we could view those messages in our dashboards. After the migration, all of it stopped. With the new versions, do we have to use Sentry.captureException? It seems like a lot of effort to migrate to newer versions. I came across this SentryAppender for the Logback but was unsure as to how to use it. 

Can someone please advise on how to continue sending events to the Sentry via logger.error which reduces the effort for the changes exponentially?

**Comment by adinauer** [comment]
Hello @ravisaluru have you already taken a look at https://docs.sentry.io/platforms/java/guides/logback/

There's an explanation on how to have errors from the log sent to Sentry.
You can configure which log level should cause an event that is sent to Sentry with `minimumEventLevel` where the default level is `ERROR`. You can also have lower level log entries attached to the events as breadcrumbs (`minimumBreadcrumbLevel`).

Does this help?
